deployments/arbitrum/usdc/migrations/1679020486_configurate_and_ens.ts

```diff
@@ -0,0 +1,321 @@
+import { BigNumber, Contract, providers } from 'ethers';
+import { expect } from 'chai';
+import { L1ToL2MessageGasEstimator } from '@arbitrum/sdk';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { calldata, exp, getConfigurationStruct, proposal } from '../../../../src/deploy';
+
+const { INFURA_KEY } = process.env;
+
+const ENSName = 'compound-community-licenses.eth';
+const ENSResolverAddress = '0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41';
+const ENSSubdomainLabel = 'v3-additional-grants';
+const ENSSubdomain = `${ENSSubdomainLabel}.${ENSName}`;
+const ENSTextRecordKey = 'v3-official-markets';
+
+export default migration('1679020486_configurate_and_ens', {
+  prepare: async (_deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+    const { utils } = ethers;
+
+    const {
+      bridgeReceiver,
+      comet,
+      cometAdmin,
+      configurator,
+      rewards,
+    } = await deploymentManager.getContracts();
+
+    const {
+      inbox,
+      l1GatewayRouter,
+      timelock,
+      governor,
+      USDC,
+      COMP,
+    } = await govDeploymentManager.getContracts();
+
+    const l1Provider = new providers.JsonRpcProvider(`https://mainnet.infura.io/v3/${INFURA_KEY}`);
+    const l2Provider = new providers.JsonRpcProvider(`https://arbitrum-mainnet.infura.io/v3/${INFURA_KEY}`);
+    const gasEstimator = new L1ToL2MessageGasEstimator(l2Provider);
+
+    const l1GatewayRouterInterace = l1GatewayRouter.interface;
+
+    const USDCAmountToBridge = exp(10_000, 6);
+    const COMPAmountToBridge = exp(2_500, 18);
+    const usdcGatewayAddress = await l1GatewayRouter.getGateway(USDC.address);
+    const compGatewayAddress = await l1GatewayRouter.getGateway(COMP.address);
+    const refundAddress = comet.address; // XXX timelock address?
```

Comment:
if there is unused gas for bridged actions, Arbitrum has you specify an L2 address that will receive the refund. Not sure if that should be Comet. maybe the timelock?

Reply:
+1 on setting this to the L2 Timelock

Reply:
updated to the L2 timelock